code/datums/quirks/good.dm

```diff
@@ -116,6 +178,52 @@
 	mood_quirk = TRUE
 	medical_record_text = "Patient demonstrates low-inhibitions for physical contact and well-developed arms. Requesting another doctor take over this case."
 
+/datum/quirk/fireproof
+	name = "Fireproof"
+	desc = "Due to training, natural aptitude, or consuming large quantities of Aerogel as a child, your skin has developed a resistance to fire."
+	icon = "fire"
+	value = 8
+	gain_text = "<span class='notice'>You feel resistant to fire!</span>"
+	lose_text = "<span class='danger'>You no longer feel resistant to fire.</span>"
+	medical_record_text = "Patient's skin shows natural resistance to fire."
+
+/datum/quirk/fireproof/add()
+	if(ishuman(quirk_holder))
+		var/mob/living/carbon/human/fire_man = quirk_holder
+		fire_man.physiology.heat_mod *= 0.25
+
+/datum/quirk/fireproof/remove()
+	if(ishuman(quirk_holder))
+		var/mob/living/carbon/human/fire_man = quirk_holder
+		fire_man.physiology.heat_mod *= 4
+
+/datum/quirk/indefatigable
+	name = "Indefatigable"
+	desc = "Through consistant exersize and good lifestyle choices you have an increased stamina pool."
+	icon = "heartbeat"
+	value = 8
+	gain_text = "<span class='notice'>You feel like you could run a mile.</span>"
+	lose_text = "<span class='danger'>You feel tired.</span>"
+	medical_record_text = "Patient has an exelent diet and exersises consistantly, well done."
+
+/datum/quirk/indefatigable/add()
+	if(ishuman(quirk_holder))
+		var/mob/living/carbon/human/subject = quirk_holder
+		subject.physiology.stamina_mod *= 0.8
```

Comment:
This one alone actually makes it so quietly, batons now require three hits and not two, with no real outside indicators that this would be the case. I can imagine that this would get blocked on this basis alone.